docs/KeyCaching.md

```diff
@@ -3,13 +3,15 @@
 Asherah supports caching of Intermediate and System Keys to minimize calls against external resource and reducing
 latency. This is especially important for minimizing calls to the configured
 [Key Management Service](KeyManagementService.md) as that is often the most expensive and latent interaction of the
-resources involved. The cache is built on top of the Secure Memory implementation.
+resources involved. The cache is built on top of the Secure Memory implementation. 
+The API also supports caching of Sessions, which further helps in reducing the latency. If 2 different users try to
```

Comment:
a couple minor suggestions:
1. i would suggest trying to highlight that session caching is primarily intended for stateless workloads that can't make use of a shared session in the calling app. it's being indirectly insinuated here but it's not as clear/direct IMO
2. in addition to reducing latency, be good to mention you are reducing API calls to the metastore (i.e. API calls/costs in a cloud environment)